src/components/CookieBar/CookieBarButtons.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const propTypes = {
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  className: PropTypes.string
```

Comment:
`classname` can be removed. You can just use
`const buttonClasses = classNames(className, 'cookiebar-buttons')` below.